packages/entities/entities-certificates/src/components/CACertificateList.cy.ts

```diff
@@ -73,10 +73,10 @@ describe('<CACertificateList />', () => {
         props: {
           cacheIdentifier: `ca-certificate-list-${uuidv4()}`,
           config: baseConfigKonnect,
-          canCreate: () => {},
-          canEdit: () => {},
-          canDelete: () => {},
-          canRetrieve: () => {},
+          canCreate: () => true,
```

Comment:
question: you're changing a ton of defaults in tests - is this desired? I would imagine this would impact the pass/fail

Reply:
Oops, I think we should change it to `() => false` here.
Most of the `canXXX`s are declared as `() => boolean | Promise<boolean>`. Before we improved the typing support for`cy.mount`, it could not detect the wrong props type so passing `() => {}` didn't trigger any error.
We should change it here but it seems like changing it to `() => false` is safer than changing it to `() => true`